dvc/remote/gdrive/google-dvc-client-id.json

```diff
@@ -0,0 +1 @@
+{"installed":{"client_id":"719861249063-v4an78j9grdtuuuqg3lnm0sugna6v3lh.apps.googleusercontent.com","project_id":"data-version-control","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"2fy_HyzSwkxkGzEken7hThXb","redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]}}
```

Comment:
Do we need a separate file for it or could we simply put this into oauth.py? This is really minor, but just don't really want to deal with additional non-code files in the packages :smile:

Reply:
Hmm. I also don't like this, and there is a method like `flow.from_client_config`. But then we would need to handle the `credentialpath` config option explicity...